### PR TITLE
Issue/ha

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

These are some changes that have been on my disk for a while, I need some of it to fix the performance test

# Merge procedure

Don't use the github built-in merge, but the process described [here](https://docs.internal.inmanta.com/topics/tasks/commiting_changes_modules.html)

```sh
git pull
git checkout master
git pull
git merge --squash issue/{issue-number}-{short description}
inmanta module commit -m "{Commit Message Here}" -r
git push
git push {tag} # push the tag as well
```

Then close the PR with a reference to the commit

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] ~~Attached issue to pull request~~
- [x] Changelog entry
- [x] Version number is bumped to dev version
- [x] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [x] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: templates/pg_hba_master.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+host    all             all             127.0.0.1/32            md5
+# IPv6 local connections:
+host    all             all             ::1/128                 md5
+# Allow replication connections from localhost, by a user with the
+# replication privilege.
+local   replication     all                                     peer
+host    replication     all             127.0.0.1/32            ident
+host    replication     all             ::1/128                 ident
+host    replication     replication     {{host.ip}}/32        md5
+host    replication     replication     {{standby.host.ip}}/32       md5
```
[the remark this answers is not in the document]
Comment (wouterdb) [reply templates/pg_hba_master.conf.j2:95]: good catch, added

File: model/_init.cf [hunk 2]
```diff
@@ -10,13 +13,26 @@ entity PostgresqlServer extends ip::services::Server:
     bool managed = true
 end
 
+entity PostgresqlTools:
+
+end
```
[the remark this answers is not in the document]
Comment (wouterdb) [reply model/_init.cf]: added

File: model/_init.cf [hunk 3]
```diff
@@ -32,11 +48,27 @@ implementation postgresqlServer for PostgresqlServer:
...
+
+implementation install_tools for PostgresqlTools:
+    postgresql_repo = yum::Repository(host=self.host,
+                                      name="postgresql-10",
+                                      gpgcheck=true,
+                                      enabled=true,
+                                      baseurl="https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-$releasever-$basearch",
+                                      gpgkey="https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG",
+                                      requires=self.requires)
+    install_postgresql_package = std::Package(name="PostgresqlTools", host=self.host, state="latest", requires=[postgresql_repo])
```
Comment (arnaudsjs) [inline model/_init.cf:62]: I don't think the `PostgresqlTools` package exists.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: model/_init.cf (hunk 3 above)
Comment (wouterdb) [reply model/_init.cf:62]: dang, what happened here. thx

**Review by arnaudsjs — APPROVED** [review]

**Comment by wouterdb** [comment]
merged in 49c7caa

**Pull request closed without merging**